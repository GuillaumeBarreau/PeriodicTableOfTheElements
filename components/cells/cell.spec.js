import { shallowMount } from '@vue/test-utils'
import cmp from '~/components/cells/cell.vue'

describe('testing component cell', () => {
  const wrapper = shallowMount(cmp, {
    propsData: {
      objectElement: {
        atomicNumber: 'atomicNumber-test',
        symbol: 'symbol-test',
        name: 'name-test',
        atomicMass: 'atomicMass-test'
      }
    }
  })

  it('snapshot testing', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })

  describe('Component Validation', () => {
    it('Should return the component cell.vue', () => {
      expect(wrapper.is(cmp)).toBe(true)
    })
  })
})
