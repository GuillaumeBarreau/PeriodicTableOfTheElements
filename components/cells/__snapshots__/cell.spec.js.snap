// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing component cell snapshot testing 1`] = `
"<div class=\\"item--cell\\"><p class=\\"data--cell--number\\">
    atomicNumber-test
  </p> <p class=\\"data--cell--symbol\\">
    symbol-test
  </p> <p class=\\"data--cell--name\\">
    name-test
  </p> <p class=\\"data--cell--mass\\">
    atomicMass-test
  </p></div>"
`;
